src/rkc_utils.jl

```diff
@@ -1,15 +1,19 @@
 # This function calculates the largest eigenvalue
 # (absolute value wise) by power iteration.
-
+const RKCAlgs = Union{RKC,IRKC}
 function maxeig!(integrator, cache::OrdinaryDiffEqConstantCache)
   isfirst = integrator.iter == 1 || integrator.u_modified
   @unpack t, dt, uprev, u, f, p, fsalfirst = integrator
   maxiter = 50
-  safe = (integrator.alg isa RKC) ? 1.0 : 1.2
+  safe = (integrator.alg isa RKCAlgs) ? 1.0 : 1.2
```

Comment:
<:

Reply:
@ChrisRackauckas 
getting this error when changing this
```
ERROR: TypeError: in <:, expected Type, got 
IRKC{0,true,LinSolveFactorize{typeof(LinearAlgebra.lu!)},
NLNewton{true,OrdinaryDiffEq.NLNewtonCache{Nothing,Nothing,Nothing,
Nothing,Nothing,Nothing}},DataType,Nothing,Nothing,Float64,Int64,:Predictive}
```

Reply:
Yes. Need to typeof